docs/content/en/docs/getting-started/optional/registrymirror.md

```diff
@@ -140,25 +139,3 @@ For [Mac](https://docs.docker.com/desktop/mac/#add-tls-certificates), you can fo
 {{% alert title="Note" color="primary" %}}
   You may need to restart Docker after adding the certificates.
 {{% /alert %}}
-
-## Registry configurations
```

Comment:
shouldnt we keep this?  if the registry they are using requires these to exist ahead of time, don't they still need to do this?

Reply:
yeah I think this section is still applicable

Reply:
Is it only required for harbor? I never created those projects for my docker registry server running locally?

Reply:
i know it's needed for harbor and i think another one as well, maybe artifactory

Reply:
yeah we should keep this.

Reply:
added back